### PR TITLE
chore: update default version

### PR DESCRIPTION
**Pull request by sapk**
I know that the version is generate in makefile but it would be good to keep it up-to-date from time to time.

Related #9913

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

File: main.go
Comment (techknowlogick) [inline main.go]: I think it would be better if we have this set to 0.0.0-dev, as we have our makefile override this, and then we could also distinguish better when the binary is built via go build without version info passedin.

**Comment by 6543** [comment]
@techknowlogick we had this topic before ... #8686

**Comment by techknowlogick** [comment]
@6543 I just don't think this should have to be kept up to date, as makefile already gives us the info. Having it be a non version would prevent users from being confused when we don't update this line.

My opinion is still the same as the other PR you made.

File: main.go
Comment (techknowlogick) [reply main.go]: or maybe have it blank like the below variables?

**Comment by sapk** [comment]
Maybe it is time for a vote ?

- 👍 empty
- 👎 updated version (or at least try)
- 😄 0.0.0-dev
- 🎉 develop
- 😕 leave as it is

(smileys are just listed by order and doesn't means anythings)

**Comment by 6543** [comment]
@go-gitea/maintainers vote: https://github.com/go-gitea/gitea/pull/9916#issuecomment-576715874

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — CHANGES_REQUESTED** [review]
blocked until vote is over

**Comment by lunny** [comment]
The version is some meaning when you download the tar zip source file. Because there is no git commit information to let you know what's the version of this source copy.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
What @lunny has said makes sense. I've taken away my block.

**Comment by jolheiser** [comment]
Based on the related issue I would actually argue against updating this value unless we plan on doing so for every release.  
I personally think this string should make it entirely clear that a build made without the link flags is **not** indicative of the actual version.

File: main.go
Comment (6543) [inline main.go]:
```suggestion
	Version = "develop"
```
as vote suggested ...

**Review by 6543 — CHANGES_REQUESTED** [review]
suggestion

**Comment by guillep2k** [comment]
How about "built from sources", "please build using make" or "don't bother, it's not relevant" 🤣 

No, really, some text that implies that it was built outside of the normal build flow.

**Comment by techknowlogick** [comment]
@guillep2k that’s why I had suggested 0.0.0-dev previously

**Comment by sapk** [comment]
The -dev flag imply that it was build without make.  
I wasn't intending to fire up such a debat (sorry) but the vote will clear this one for all and it seems to be "develop" the winner for now.

I think it is good to close some old debat that was put aside and clear them by a vote by maintainers and not stay with a status quo.

File: main.go
Comment (guillep2k) [inline main.go]: Sorry, but we might just consider...
```suggestion
	Version = "development"
```

**Comment by davidsvantesson** [comment]
@guillep2k Maybe set to something ironic, like "did-you-make-it?"

File: main.go
Comment (sapk) [reply main.go]: I added it to the choices. We can wait a little more to choose.

**Comment by tboerger** [comment]
For drone plugins im setting it to unknown by default

**Comment by 6543** [comment]
@sapk can you add "did-you-make-it" to the vode :D

**Comment by 6543** [comment]
@sapk is core not chore or ?

**Comment by sapk** [comment]
I used `chore:` for a non essential code change like https://www.conventionalcommits.org/en/v1.0.0/

**Review by 6543 — APPROVED** [review]

**Comment by wxiaoguang** [comment]
I believe we need to set AppVer to valid version string now.

See:

* https://github.com/go-gitea/gitea/pull/18146
* #18124 
* https://github.com/go-gitea/gitea/pull/18124#pullrequestreview-842261656

Or, is there any better way to avoid AppVer appearing in many unit tests?

**Pull request merged**
